runtime/oti/j9nonbuilder.h

```diff
@@ -1641,7 +1641,7 @@ typedef struct J9MethodParametersData {
 	J9MethodParameter parameters;
 } J9MethodParametersData;
 
-#if defined(__xlC__) || defined(__ibmxl__) || defined(__open_xl__) || defined(__GNUC__) || defined(_MSC_VER) || defined(J9ZOS390) || defined(LINUX) || defined(AIXPPC) || defined(WIN32)
+#if defined(__xlC__) || defined(__ibmxl__) || defined(__GNUC__) || defined(_MSC_VER) || defined(J9ZOS390) || defined(LINUX) || defined(AIXPPC) || defined(WIN32)
```

Comment:
You'll need to be careful when rebasing. You've accidentally included reverts to recent patches that shouldn't be reverted. Please make sure that only the code that you are wanting to change is included in the PR.